src/style.css

```
@@ -0,0 +1,136 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+p {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto 40px;
+}
+
+.card__discription {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.stars__rev {
+  color: #060b35;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 17px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.stars__star {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(-n+1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
```

if you have the same property values for many selectors, you can write them separated by a comma 
```suggestion
.stars--1 :nth-child(-n+1), 
.stars--2 :nth-child(-n+2),
..... {
  background-image: url(images/star-active.svg);
}

```